code/modules/mob/living/emote.dm

```diff
@@ -594,4 +594,6 @@
 
 /datum/emote/living/hiss/get_sound(mob/living/user)
 	if(islizard(user))
-		return pick('sound/voice/hiss1.ogg', 'sound/voice/hiss2.ogg', 'sound/voice/hiss3.ogg', 'sound/voice/hiss4.ogg', 'sound/voice/hiss5.ogg', 'sound/voice/hiss6.ogg')
+		return 'nsv13/sound/voice/lizard/liz_hiss.ogg'
+	if(iscatperson(user))
```

Comment:
Add NSV tags for this block please

Reply:
It'd probably be easier to just override this proc in the modular file